examples/Microsoft.Spark.CSharp.Examples/Sql/Batch/readmes/GitHubProjectsReadme.md

```diff
@@ -16,7 +16,7 @@ or changing.
 
 The data used in this example was generated from [GHTorrent](http://ghtorrent.org/), which monitors all public GitHub events (such as info about projects, commits, and watchers), stores the events and their structure in databases, and then releases data collected over different time periods as downloadable archives. 
 
-The dataset used when creating this sample was [downloaded from the GHTorrent archives](http://ghtorrent.org/downloads.html). Specifically, the **projects.csv** file was extracted from one of the latest MySQL dumps. For analysis that only takes a few seconds in demos, projects.csv was shortened to only a few GB, and thus the dataset is called **projects_smaller.csv** throughout this sample.
+The dataset used when creating this sample was [downloaded from the GHTorrent archives](http://ghtorrent.org/downloads.html). Specifically, the **projects.csv** file was extracted from one of the latest MySQL dumps. For analysis that only takes a few seconds in demos, projects.csv was shortened to only a few GB, and thus the dataset is called **[projects_smaller.csv](../projects_smaller.csv)** throughout this sample.
```

Comment:
This paragraph mentions projects_smaller.csv being a few GB.  The file uploaded is not a few GB.

Reply:
Good catch, changed.